src/store/getters.js

```diff
@@ -133,6 +133,9 @@ export default {
       || categorizeContractCallTxObject(transaction)?.url
       || ''
   ),
+  getPayload: () => (transaction) => (transaction.tx?.payload)
+    ? TxBuilderHelper.decode(transaction.tx?.payload).toString()
+    : null,
```

Comment:
From what I see this getter is not taking anything from the store, so In my opinion this is more like a utility function, not a store getter.